extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -1,25 +1,71 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
+use Wikia\Logger\WikiaLogger;
 
 class ImageReviewSpecialController extends WikiaSpecialPageController {
+
 	const ACTION_QUESTIONABLE = 'questionable';
 	const ACTION_REJECTED     = 'rejected';
 	const ACTION_UNREVIEWED   = 'unreviewed';
-
-	private $statsHeaders = array( 'user', 'total reviewed', 'approved', 'deleted', 'qustionable', 'distance to avg.' );
+	const ACTION_STATS   = 'stats';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "stats" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)